.github/workflows/turbo-check.yml

```
@@ -118,7 +118,7 @@ jobs:
 
           # Add trigger information
           if [ "${{ github.event_name }}" == "pull_request" ]; then
-            echo "**Trigger:** Pull Request #${{ github.event.number }} against \`${{ github.base_ref }}\`" >> $GITHUB_STEP_SUMMARY
+            echo "**Trigger:** Pull Request #${{ github.event.number }} against \`${{ github.base_ref }}\`" >> $GITHUB_STEP_SUMMARY                                             
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)